php/Terminus/Helpers/UpdateHelper.php

```diff
@@ -17,11 +17,13 @@ class UpdateHelper extends TerminusHelper
     public function getCurrentVersion()
     {
         $request  = new Request();
-        $url      = 'https://api.github.com/repos/pantheon-systems/terminus/releases';
-        $url     .= '?per_page=1';
-        $response = $request->request($url, ['absolute_url' => true]);
-        $release  = array_shift($response['data']);
-        $cache    = new FileCache();
+        $url = 'https://api.github.com/repos/pantheon-systems/terminus/releases/tags/0.x?per_page=1';
```

Comment:
If my comment above is not clear you can checkout the difference between these release api calls:
https://api.github.com/repos/ajbarry/jetbrains-settings/releases/latest
https://api.github.com/repos/ajbarry/jetbrains-settings/releases?per_page=1